lib/gems/pending/util/object_storage/miq_swift_storage.rb

```diff
@@ -0,0 +1,148 @@
+require 'pp'
+# require 'util/miq_object_storage'
```

Comment:
I might have left this in as a comment at one point, but I think you can delete this.

Actually... looking at this more, possibly this should change to use this line... let me get back to you on that one.

Reply:
@NickLaMuro let me know...

Reply:
I think I might have been trying to test this file in `irb` on it's own, and doing the below require made it easier to do that.  I will give this a shot in a few with this uncommented and the other included and see if it should work.

I am pretty sure this line should be the one we want, but I want to make sure tests pass (unit and integration) before I say one way or the other.